cmd/snap/cmd_snap_op_test.go

```diff
@@ -39,6 +39,8 @@ type snapOpTestServer struct {
 	total   int
 }
 
+var _ = check.Suite(&SnapOpSuite{})
```

Comment:
probably the only reason i'd disrecommend `go-check` is because of this kind of boilerplate. Is there no better way, @niemeyer ?